apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx

```diff
@@ -665,55 +668,46 @@ export const PoolPoster = ({
-                    "Do you require accommodations, or do you have any questions about this process?",
-                  id: "2K8q04",
-                  description:
-                    "Opening sentence asking if accommodations are needed",
-                })}
-              </Text>
-              <Text>
-                {intl.formatMessage({
-                  defaultMessage:
-                    "Please contact the Digital Community Management team if you require any accommodations during this application process.",
-                  id: "p3j/0q",
-                  description:
-                    "Description of what to do when accommodations are needed",
-                })}
-              </Text>
-              {contactEmail && (
+            {contactEmail && (
+              <TableOfContents.Section id={sections.contact.id}>
+                <TableOfContents.Heading>
+                  {sections.contact.title}
+                </TableOfContents.Heading>
                 <Text>
                   {intl.formatMessage(
                     {
                       defaultMessage:
-                        "<strong>Email</strong>: <anchorTag>{emailAddress}</anchorTag>",
-                      id: "Wnw+oz",
-                      description: "An email address to contact for help",
+                        "Please contact the <a>{name} team</a> by email if you have <strong>any questions</strong> or <strong>require an accommodation</strong> during this process.",
+                      id: "rKUVdL",
+                      description:
+                        "Opening sentence asking if accommodations are needed",
                     },
                     {
-                      anchorTag: (chunks: React.ReactNode) =>
+                      a: (chunks: React.ReactNode) =>
                         anchorTag(chunks, contactEmail),
-                      emailAddress: contactEmail,
+                      name: getLocalizedName(pool.team?.displayName, intl),
                     },
                   )}
                 </Text>
-              )}
-            </TableOfContents.Section>
-            <TableOfContents.Section id={sections.hiringPolicies.id}>
+              </TableOfContents.Section>
+            )}
+            <TableOfContents.Section id={sections.whoCanApply.id}>
               <TableOfContents.Heading>
-                {sections.hiringPolicies.title}
+                {sections.whoCanApply.title}
               </TableOfContents.Heading>
               <Text>
                 {intl.formatMessage({
                   defaultMessage:
-                    "Preference will be given to veterans, Canadian citizens and to permanent residents.",
-                  id: "IF1xj8",
+                    "Persons residing in Canada, and Canadian citizens and Permanent residents abroad.",
+                  id: "DomrM8",
+                  description: "List of criteria needed in order to apply",
+                })}
```

Comment:
Should the casing vary like this? This is the same as in the issue but is it a typo?

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/5cb84fa1-e753-481d-9769-db8a72787517)

Reply:
Hmm, yeah that makes sense. @mnigh Any idea how we would prefer this to be cased?

Reply:
permanent resident should be sentence case, so all lower case for these examples.

Reply:
Thanks, updated in [fix case inconsistency](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7287/commits/8e20bbc97c4f4ecbadf3a015d1fb97a21fd7938c)